src/Libraries/DSIronPython/DSIronPython.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\packages\Microsoft.CodeCoverage.16.2.0\build\netstandard1.0\Microsoft.CodeCoverage.props" Condition="Exists('..\..\packages\Microsoft.CodeCoverage.16.2.0\build\netstandard1.0\Microsoft.CodeCoverage.props')" />
```

Comment:
did we import these props and targets previously?

Reply:
Nope, I can totally remove them if you prefer

Reply:
yeah, lets make minimal changes to the build - we just want this dll to show up but not effect anything else..  - I'm not sure it's possible to add comments to a csproj but if possible that might also be good (IE - TODO remove this!)

Reply:
@mjkkirschner TODO comments for removal are added below already, I will remove these extra parts